lib/components/WheelPicker/index.js

```diff
@@ -2,11 +2,26 @@ import _ from 'lodash';
 import PropTypes from 'prop-types';
 import React, {Component} from 'react';
 import {DeviceEventEmitter, requireNativeComponent, StyleSheet, View} from 'react-native';
-import {Picker} from '@react-native-community/picker';
+
 import WheelPickerItem from './WheelPickerItem';
 import {Constants} from '../../../src/helpers';
 import {Typography, Colors} from '../../../src/style';
 
+/* Safely require picker component */
+let Picker = () => null;
+try {
+  Picker = require('@react-native-picker/picker').Picker; // New package
```

Comment:
It seems that the metro (I assume) knows to take `@react-native-community/picker` when requiring `@react-native-picker/picker` (I did not clean-install and it works for me), do we want to keep this just in case?

Reply:
Yes I noticed that as well, But I kept it just in case.